notebooks/official/tabnet/tabnet_vertex_tutorial.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>Consider pulling out some of these variables into constants, especially those used later in the notebook during the BQ input stage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>I think define all possible constants already.</p><p>If you mean the the value in config like below, it should not be defined as constant.</p><p>"""studySpec:</p><p>&nbsp;metrics:</p><p>&nbsp;- metricId: auc</p><p>&nbsp;&nbsp;goal: MAXIMIZE</p><p>&nbsp;parameters:</p>